Microsoft.WindowsAzure.Storage/CMakeLists.txt

```diff
@@ -15,7 +15,33 @@ option(BUILD_SAMPLES "Build sample codes" OFF)
 if(UNIX)
   find_package(Boost REQUIRED COMPONENTS log log_setup random system thread locale regex filesystem chrono date_time)
   find_package(Threads REQUIRED)
-  find_package(OpenSSL REQUIRED)
+  if(APPLE AND NOT OPENSSL_ROOT_DIR)
+    # Prefer a homebrew version of OpenSSL over the one in /usr/lib
```

Comment:
How about use `brew link --force openssl`?  This command forcefully link all openssl libraries and headers to `/usr/local/`.  Then we don't have to specify `OPENSSL_ROOT_DIR` at all.  And `brew link --force gettext` can do the same for `GETTEXT_LIB_DIR`.  Any concern on using this command `brew link --force`?